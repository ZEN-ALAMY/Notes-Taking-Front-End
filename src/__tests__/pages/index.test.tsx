import { render } from "@testing-library/react";
import Index from "../../pages/index";

describe("Index page tests", () => {
  it("should render the UI correctly", () => {
    expect(render(<Index />).asFragment()).toMatchSnapshot();
  });
});
