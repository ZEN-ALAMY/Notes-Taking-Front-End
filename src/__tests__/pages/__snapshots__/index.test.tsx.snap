// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page tests should render the UI correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-green-500 sm:bg-red-500 xl:bg-blue-500 h-[21%] flex flex-col  items-center justify-evenly text-black font-mono font-black "
  >
    <h1
      class="text-3xl sm:text-4xl xl:text-5xl font-extrabold"
    >
      NOTES TAKING APP
    </h1>
    <div
      class="flex flex-row gap-x-4 xl:gap-x-8 sm:gap-x-6"
    >
      <button
        class="bg-slate-100 rounded-xl border-none xl:w-[100px] sm:w-[80px] w-[60px]"
      >
        <span
          class="text-sm sm:text-lg xl:text-2xl"
        >
          HOME
        </span>
      </button>
      <button
        class="bg-slate-100 rounded-xl border-none xl:w-[100px] sm:w-[80px] w-[60px]"
      >
        <span
          class="text-sm sm:text-lg xl:text-2xl"
        >
          ALL
        </span>
      </button>
      <button
        class="bg-slate-100 rounded-xl border-none xl:w-[100px] sm:w-[80px] w-[60px]"
      >
        <span
          class="text-sm sm:text-lg xl:text-2xl"
        >
          ADD
        </span>
      </button>
      <button
        class="bg-slate-100 rounded-xl border-none xl:w-[100px] sm:w-[80px] w-[60px]"
      >
        <span
          class="text-sm sm:text-lg xl:text-2xl"
        >
          REMOVE
        </span>
      </button>
    </div>
  </div>
  <div
    class="h-screen"
  >
    <div
      class="flex flex-row items-center h-[85%] justify-center"
    >
      <span
        style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
      >
        <span
          style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27400%27%20height=%27400%27/%3e"
            style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
          />
        </span>
        <img
          alt="hello"
          data-nimg="intrinsic"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        />
        <noscript />
      </span>
    </div>
  </div>
</DocumentFragment>
`;
